sprint4

MuzAnSka opened the pull request:
Project work of sprint 4, Anna Kapaneva

MuzAnSka committed:
sprint4

egorvas commented on pom.xml:
```diff
@@ -0,0 +1,60 @@
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
```
**Нужно исправить:** у тебя junit указан 5 раз, а должен быть один

egorvas commented on src/test/java/DropDownListParamTest.java:
```diff
@@ -0,0 +1,51 @@
+    @Parameterized.Parameters
+    public static Object[][] expectedAnswersParamList() {
+        return new Object[][]{
+                {"accordion__heading-0", "accordion__panel-0", "Сутки — 400 рублей. Оплата курьеру — наличными или картой."},
```
**Нужно исправить:** по условиям задания локаторы должны быть вынесены в PO

egorvas commented on src/test/java/DropDownListTest.java:
```diff
@@ -0,0 +1,33 @@
+
+    // Тест соответствия текста в выпадающем списке
+    @Test
+    public void CheckDropDownListText() {
```
**Нужно исправить:** функции java всегда с маленькой буквы

egorvas commented on src/test/java/DropDownListTest.java:
```diff
@@ -0,0 +1,33 @@
+    @Test
+    public void CheckDropDownListText() {
+
+        new MainPage(driver)
```
**Нужно исправить:** я не понимаю зачем тебе два теста, да еще и в разных классах. Объедини логику(с dropDownListTest), оставь один тест, причем параметризованный.

egorvas commented on src/test/java/SamokatOrderingTest.java:
```diff
@@ -0,0 +1,63 @@
+
+// Тест заказа через кнопку в хэдере
+    @Test
+    public void samokatOrderingByHeaderOrderButton() {
```
**Можно лучше:** у тебя два фактически одинаковых теста, рекомендую использовать параметризацию и оставить только один

MuzAnSka committed:
Исправлено:
- у тебя junit указан 5 раз, а должен быть один
- по условиям задания локаторы должны быть вынесены в PO (внесено в mainPage)
- функции java всегда с маленькой буквы
- Объедини логику(с dropDownListTest), оставь один тест, причем параметризованный.

Я не знаю как убрать файл DropDownListTest.java, поэтому попытаюсь его убрать в ручную, ну или у меня не получиться.

MuzAnSka committed:
Delete src/test/java/DropDownListTest.java

Удаление ненужного файла
